web/app/scripts/compiled-es5/directives/form.js

```diff
@@ -28,37 +28,39 @@ _module.directive('surveysForm', [_api2.default.componentName, _fieldsetsParser2
         restrict: 'A',
         scope: {
             formUrl: '=',
-            responseUrl: '='
+            responseUrl: '=',
+            onSuccess: '&',
+            onFailure: '&'
         },
         templateUrl: 'templates/incuna-surveys/form/survey-form.html',
-        link: function link($scope) {
```

Comment:
I understood `$scope` to be something that Angular injected `.controller('someCtrl' ['$scope', function ($scope) { ... }])` so it seems confusing to use it the `link` method - not sure if it any different.